cluster/node-pools/master-default/userdata.yaml

```diff
@@ -865,3 +869,28 @@ write_files:
       {{- if eq .Cluster.ConfigItems.enable_encryption "true" }}
           - identity: {}
       {{- end }}
+
+{{ if eq .Cluster.ConfigItems.event_rate_limit_enable "true"}}
+  # Enabling EventRateLimit admission plugin
+  # Reference: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#eventratelimit
+  - owner: root:root
+    path: /etc/kubernetes/config/admission-config.yaml
+    permission: '0400'
```

Comment:
Let's drop the strict permissions here like in most other files. The only one that has it is encryption related so it might be ok there but here we can remove it.

Reply:
Alright!

Reply:
Done!